### ISSUE TITLE
⌨ Assessment dialog not working for Dragon Naturally Speaking

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#16049]
### 🛡️Fable tester reported: 
When assessing candidates:
Number 1: after selecting "to assess" I cannot select any of the buttons within the dialog using the commands "click" followed by the name of the button or the command "click button".
Number 2: when the dialog is open and I used the command "click button" it only shows me buttons behind the dialog. This only occurs after I had filled out the first two requirements.
Number 3: some of the options are only given to me if I use the mouse grid command to manually select "demonstrated" when I have a dialog open. When using the command "click demonstrated" it gives me different options.

Due to all of these issues I could only use the mouse grid command for many elements which takes a lot of time to manually select buttons and links and is extremely frustrating.

### Expected behaviour: 
you can interact with the buttons inside the dialog by using the voice command "Click" followed by the name or the command "click button" 
You shouldn't be able to interact with elements behind the dialog when it's open. 

### WCAG 2.1 Criteria

2.5.3 Label in Name

### 📋 Steps to Reproduce

1. sign in 
2. change to community view 
3. try to [complete any of the assessments of this candidate](https://uat-talentcloud.tbs-sct.gc.ca/en/admin/candidates/9badd676-d04b-40ae-a8ad-7093239f7ceb/application) using voice commands 

### What operating systems are you encountering the problem on?

Windows

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

Dragon Naturally Speaking

### ✅ Acceptance Criteria

- [ ] Voice commands work for items inside the assessment dialog when open 
- [ ] You cannot interact with elements outside of the dialog when open
